mappings/net/minecraft/entity/ExperienceOrbEntity.mapping

```diff
@@ -27,10 +30,11 @@ CLASS net/minecraft/class_1303 net/minecraft/entity/ExperienceOrbEntity
 		ARG 2 amount
 	METHOD method_31497 merge (Lnet/minecraft/class_1303;)V
 		ARG 1 other
-	METHOD method_31498 onEverySecond ()V
-		COMMENT Called every second (every 20 ticks).
+	METHOD method_31498 expensiveUpdate ()V
```

Comment:
What does this method actually do? The name and most of the documentation are not helpful in describing the actual implementation of the method.

Reply:
It performs checks and updates that are not as critical to each tick, such as moving to player, merging.